include/deal.II/fe/fe_raviart_thomas.h

```diff
@@ -93,7 +93,7 @@ DEAL_II_NAMESPACE_OPEN
  * quadrature rules themselves. The generalized support points are a set of
  * points such that this quadrature can be performed with sufficient accuracy.
  * The points needed are those of QGauss<sub>k+1</sub> on each face as well as
- * QGauss<sub>k</sub> in the interior of the cell (or none for
+ * QGauss<sub>k+1</sub> in the interior of the cell (or none for
  * RT<sub>0</sub>).
```

Comment:
Maybe it would clearer why this is correct if you gave an example. Let's do 2d: Think about the RT(0) case. There, you have to have 1 point per edge, but none in the interior. That would correspond to a QGauss(0) in the interior, as also mentioned in parentheses.

Likewise, if you had a RT(1), you'd have 2 points per edge, and 4 points in the interior (2 points for each of the vector components of element, of which there are 2 in 2d). These can be combined to a QGauss(2) in the interior.

For RT(3), there are 3 points per edge, so we need 6 points per vector components in the interior for a total of 12 interior quadrature points. This, however, is not equal to the number of quadrature points for a QGauss(3) any more, which has 9 of these points.

So I'm not quite sure any more what is correct...

Reply:
For this space, the number of generalized support points which are obtained from these QGauss rules, are not equal to the number of dofs per cell. We need more QGauss points in interior to compute moments exactly. So it all looks correct to me.

Reply:
The cell moments are wrt test functions which are of lower degree than the RT shape functions.

Reply:
I see. Thanks for the explanation!